packages/zoe/test/swingsetTests/zoe/test-zoe.js

```diff
@@ -285,6 +286,26 @@ test.serial('zoe - sellTickets - valid inputs', async t => {
   t.deepEqual(dump.log, expectedSellTicketsOkLog);
 });
 
+const expectedOTCDeskOkLog = [
+  '=> alice, bob, carol and dave are set up',
+  'Inventory added',
+  '@@ schedule task for:1, currently: 0 @@',
+  'Trade successful',
+  '{"brand":{},"value":3}',
+  '{"brand":{},"value":0}',
+  'Inventory removed',
+  '{"brand":{},"value":2}',
+];
+test.serial('zoe - otcDesk - valid inputs', async t => {
```

Comment:
Why does it need to be serial? I suspect we should document these annotations when we add them.

Reply:
This was done by Brian in this commit. My test does not do anything special here: https://github.com/Agoric/agoric-sdk/commit/c5364b8be273e3e307d8f5d4ef947047a725c702